app/views/users/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<img src="/Users/arch/LaunchAcademy/challenges/ruby_on_trails/public/uploads/user/profile_photo/1/pic.jpg" alt="">
```

Comment:
It looks like you're actually just hard-coding one image in here, when really you should be using your user's image. Additionally, if you DO want to use this image, you're currently hardcoding the path on your OWN machine, which will fail on any of your teammate's machines and in production. Instead, you'd want to use the `img_tag` helper: https://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html#method-i-image_path